exercicios/para-casa/entregas/larissa-gnandt/resolucao.js

```diff
@@ -0,0 +1,48 @@
+class ContaBancaria {
+  constructor(titular, saldo, limiteDaConta) {
+    this.titular = titular;
+    this.saldo = saldo;
+    this.limiteDaConta = limiteDaConta;
+    this.limiteAtivo = true;
+  }
+
+  consultarSaldo() {
+    return this.saldo;
+  }
+
+  realizarSaque(valor) {
+    if (valor > this.saldo) {
+      if (this.limiteAtivo == false) {
```

Comment:
Achei interessante a escolha do '=='. Você tentou o '==='? Fiquei curiosa quanto à possibilidade de um resultado diferente.

Reply:
Na realidade eu sempre esqueço de colocar "===", preciso me forçar a lembrar, porque no javaScript é importante né, acho que interfere sim no resultado.